<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalcWindow</class>
 <widget class="QMainWindow" name="DepositCalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>531</width>
      <height>241</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-family:'-apple-system','system-ui','Segoe WPC','Segoe UI','system-ui','Ubuntu','Droid Sans','sans-serif'; font-size:14px; color:#d4d4d4;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px;&quot;&gt;Вход: сумма вклада, срок размещения, процентная ставка, налоговая ставка, периодичность выплат, капитализация процентов, список пополнений, список частичных снятий&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-family:'-apple-system','system-ui','Segoe WPC','Segoe UI','system-ui','Ubuntu','Droid Sans','sans-serif'; font-size:14px; color:#d4d4d4;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px;&quot;&gt;Выход: начисленные проценты, сумма налога, сумма на вкладе к концу срока&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-family:'-apple-system','system-ui','Segoe WPC','Segoe UI','system-ui','Ubuntu','Droid Sans','sans-serif'; font-size:14px; color:#d4d4d4;&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/li&gt;
&lt;li style=&quot; font-family:'-apple-system','system-ui','Segoe WPC','Segoe UI','system-ui','Ubuntu','Droid Sans','sans-serif'; font-size:14px; color:#d4d4d4;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px;&quot;&gt;Input: deposit amount, deposit term, interest rate, tax rate, periodicity of payments, capitalization of interest, replenishments list, partial withdrawals list&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-family:'-apple-system','system-ui','Segoe WPC','Segoe UI','system-ui','Ubuntu','Droid Sans','sans-serif'; font-size:14px; color:#d4d4d4;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px;&quot;&gt;Output: accrued interest, tax amount, deposit amount by the end of the term&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_rate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_amount_by_the_end">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>690</y>
      <width>221</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Deposit amount by the end of the term</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_currency_mode">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>140</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>RUB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_term_mode">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>140</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>years</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_deposit_accrued_interest">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>570</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Accrued interest</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_credit_total_payment">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>690</y>
      <width>350</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_amount">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>20</y>
      <width>221</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Deposit amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_credit_overpayment">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>620</y>
      <width>350</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_monthly_payment">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>570</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_term">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>70</y>
      <width>161</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Deposit term</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_percent">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_peroidicity">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>220</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Periodicity of payments</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_credit_term">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tax_amount">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>620</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Tax amount</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_rate">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>120</y>
      <width>171</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Deposit rate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_total_amount">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_percent_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tax_rate">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>170</y>
      <width>171</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Tax rate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_tax_rate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_periodicity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>235</y>
      <width>350</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtCurrent</enum>
    </property>
    <item>
     <property name="text">
      <string>once a month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>once a quarter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>semiannually</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>once a year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>at the end of the term</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>285</y>
      <width>300</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Capitalization of interests</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_replenishment_list">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>217</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <row/>
    <column>
     <property name="text">
      <string>month</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>amount</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_withdrawals_list">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>217</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <row/>
    <column>
     <property name="text">
      <string>month</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>amount</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
